src/baseline.py

```diff
@@ -0,0 +1,72 @@
+import pandas as pd
+import numpy as np
+import pickle
+
+from sklearn.linear_model import Ridge
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import mean_squared_error
+
+rndseed = 42
+
+if __name__ == "__main__":
+
+    # import data
+    df = pd.read_csv("resources/data/master.csv")
+
+    # define vector of dependend variable
+    y = df.sale_price
+
+    # define matrix of features
+    X = df[[i for i in df.columns if i != "sale_price"]]
+
```

Comment:
Is there a strategy for how to include the time? Example: first 3 years as training, next 1 year as test and latest year as production data